.github/mistaken-pull-closer.yml

```
@@ -0,0 +1,16 @@
+# The JSONPath filter expression used to identify which PRs to close.
+# The data filtered is the pull request data along with other metadata passed in
+# by probot.
+# Default behavior: Close all PRs.
+filters:
+  - true
+
+# The message to post to the closed PR.
+commentBody: |
+  Thanks for your contribution!  Unfortunately, we don't use GitHub pull
+  requests to manage code contributions to this repository.  Instead, please
+  see [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) which provides full
```

This relative path might not be correct relative to the pull request page. It might be best to make this the full url
```suggestion
  see [CONTRIBUTING.md](https://github.com/googleapis/googleapis/blob/master/CONTRIBUTING.md) which provides full
```

---

Also, the CONTRIBUTING.md doesn't really say what to do instead of opening a pull request here.

---

Should we say this repo doesn't accept contributions? Any other suggestions?